frontend/lib/about/about_page.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:package_info/package_info.dart';
+
+import 'content_tile.dart';
+import 'texts.dart';
+
+class AboutPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+        child: FutureBuilder<PackageInfo>(
+            future: PackageInfo.fromPlatform(),
+            builder: (context, snapshot) {
+              List<Widget> children;
+              if (snapshot.hasData) {
+                children = [
+                  Center(
+                      child: Padding(
+                          padding: EdgeInsets.all(10),
+                          child: ClipRRect(
+                              borderRadius: BorderRadius.circular(20.0),
+                              child: Image(
+                                image: AssetImage("assets/icon/icon.png"),
+                                height: 100.0,
+                                width: 100.0,
+                                fit: BoxFit.cover,
+                              )))),
+                  Center(
+                      child: Text(snapshot.data.appName,
+                          style: Theme.of(context).textTheme.headline5)),
+                  Center(
+                      child: Text(snapshot.data.version,
+                          style: Theme.of(context).textTheme.bodyText2)),
```

Comment:
I think it would be nice if this section had some more space (aka padding) to breath. On my device,  it feels like the logo and the "Ehrenamtskarte" label would almost stick to the top where the bottom is pretty much empty.

Reply:
@maxammann Do you still work on this or did you reject my suggestion ? :D